src/regressionmodel.jl

```diff
@@ -53,15 +48,20 @@ The approach for computing effect is based on the effects plots described here:
 Fox, John (2003). Effect Displays in R for Generalised Linear Models.
 Journal of Statistical Software. Vol. 8, No. 15
 """
+# TODO explain that can still use the transformation method themselves
 function effects!(reference_grid::DataFrame, model::RegressionModel;
-                  eff_col=nothing, err_col=:err, typical=mean)
+                  eff_col=nothing, err_col=:err, typical=mean, invlink=identity)
     # right now this is written for a RegressionModel and implicitly assumes
-    # no link function and the existence of an appropriate formula method
     form = formula(model)
     form_typical = typify(reference_grid, form, modelmatrix(model); typical=typical)
     X = modelcols(form_typical, reference_grid)
     eff = X * coef(model)
     err = sqrt.(diag(X * vcov(model) * X'))
+    if invlink !== identity
+        ∇invlink = ForwardDiff.derivative.(invlink, eff)
+        err = err .* ∇invlink
+        eff = invlink.(eff)
```

Comment:
```suggestion
        err .*= ∇invlink
        eff .= invlink.(eff)
```
Any reason not to overwrite the arrays rather than redefine the bindings with new ones?